### ISSUE TITLE
No accounts in From dropdown at contract creation page

### ISSUE THREAD

**Issue by PatchRanger** [ethereum/mist#2938]
``````
Version: `0.9.0`
OS & Version: Ubuntu 16.04
Node type: `geth`
Number of blocks synchronized: all (4 155 467 at the moment)
Sync mode: light
``````

### Problem
I can't create any new contract as there is no account available to choose from in From field (see screenshot, the field `ОТ КОГО` means `FROM`).
There are several errors in Mist console - they could be related (refer to the screenshot).
At the same time, accounts are listed at the Wallets page. 

![image](https://user-images.githubusercontent.com/3889569/29258953-7de2208c-80e8-11e7-8d22-06e6052ba905.png)

**Comment by evertonfraga** [comment]
Please note that the Light mode is not yet fully supported. But I'll take a look at it when I get the time.

Thanks for reporting :)

**Comment by PatchRanger** [comment]
> Please note that the Light mode is not yet fully supported.

@evertonfraga Yes, I understand. I was using light-mode at my own risk - because it is such a valuable functionality. I did a quick walk through the issue queue in order to link all the related to light-mode issues together (see #2372). I hope it will help to enlight the importance of that feature request.

> But I'll take a look at it when I get the time.

Thank you for your effort!

**Comment by PatchRanger** [comment]
I've managed to establish Geth node at remote server with full blockchain and to connect to it via RPC. After having the #2959 and applying the workaround from https://github.com/ethereum/mist/issues/2959#issuecomment-327026535 , I have exact the same issue as described above for light-mode. So, I guess, it's not related to light-mode but to RPC-connection.

Correction: at Wallets page is listed only `Main account (Etherbase)` - with zero funds in it, which is wrong as I see balance in it using `browser-solidity`.

**Comment by evertonfraga** [comment]
Alright, so that means there was some sort of freeze. And as the balance "was" 0, it wouldn't show up on the From drop down.
